### DISCUSSION TITLE
Vortex compilation on arm64

### DISCUSSION THREAD

**Discussion by RedstoneWizard08** [stoatchat/discussions#289] [Q&A]
Hello. I am trying to compile Vortex on arm64 (as a docker image) and it fails with this error: `error: failed to build archive: 'libmediasoup-worker.a.p/src_lib.cpp.o': No such file or directory`

The command to build I am using is: `docker build -t revolt-vortex .`
Can you help me?

**Comment by phuocthanhlamnguyen-gif** [comment]
Check the Docker dependencies; limit the use of ninja/meson and clear the Docker build cache, if something's wrong, you can tell me
